openwisp_radius/api/views.py

```diff
@@ -419,7 +419,12 @@ def post(self, request, *args, **kwargs):
 
 class DispatchOrgMixin(object):
     def dispatch(self, *args, **kwargs):
-        self.organization = get_object_or_404(Organization, slug=kwargs['slug'])
+        try:
+            self.organization = Organization.objects.select_related(
+                'radius_settings'
+            ).get(slug=kwargs['slug'])
+        except Organization.DoesNotExist:
+            raise Http404("No Organization matches the given query.")
```

Comment:
spotted one minor issue, please use single quotes for consistency

Reply:
Done